### ISSUE TITLE
Problem z podłączeniem nucleo //serial!

### ISSUE THREAD

**Issue by malgorzatagora** [malgorzatagora/Magiczne-Schody#35]
![image](https://user-images.githubusercontent.com/9518442/45518660-9a902600-b7b2-11e8-9e5a-4329894b63b7.png)

**Comment by malgorzatagora** [comment]
Chyba podłączone mam dobrze, bo program się wgrywa...

**Comment by PawelOwsikowski** [comment]
https://www.st.com/en/development-tools/stsw-link009.html

**Comment by PawelOwsikowski** [comment]
To może rozwiązać problem:)

**Comment by Mateusz27** [comment]
@PawelOwsikowski Dokładnie, ten sterownik powinien poskutkować pojawieniem się nucleo jako portu COM.
@malgorzatagora Daj znać czy ten sterownik poprawił sytuację:)
